scipy/spatial/transform/_rotation.pyx

```diff
@@ -115,10 +115,72 @@ cdef inline void _quat_canonical(double[:, :] q) noexcept:
     for ind in range(n):
         _quat_canonical_single(q[ind])
 
+@cython.boundscheck(False)
+@cython.wraparound(False)
+cdef inline void _get_angles(
+    double[:] _angles, bint extrinsic, bint symmetric, bint sign,
```

Comment:
This leading underscore is a questionable choice in general and here it doesn't look necessary.

Reply:
Changed!